Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
 	OutputLine(NULL, "…『鬼隠し』でも、今回の『綿流し』でも。",
 		   NULL, " ...Both last time in Onikakushi, and this time in Watanagashi.", Line_WaitForInput);
-	PlaySE(4, "S02/00/440900118_2", 128, 64);
+	PlaySE(4, "S02/00/440900119", 128, 64);
```

Comment:
I'm sorry it is supposed to be S20/09/440900119.

As for 440900118_2, it's here: https://github.com/higurashi-mod/watanagashi/tree/master/SE/s02/00